guides/common/modules/proc_filtering-and-removing-facts.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+
+[id="proc_optimizing-performance-by-removing-nics-from-database_{context}"]
+= Optimizing performance by removing NICs from database
+
+Filter and exclude the connections using the `Exclude pattern for facts stored in {Project}` and  `Ignore interfaces with matching identifier` option.
+By default, these options are set to a reasonable list for hypervisors such as, `libvirt`, `ovirt`, `RHV`, or `VMWare`.
+If you name the virtual interfaces differently, you may update this filter to use it according to your requirements.
+
+.Procedure
+
+. Login to your {Project} instance.
+
+.. Open *Administer* > *Settings* > *Facts*.
```

Comment:
This is not how to phrase that. Please look for an example in the built docs/source file.